static/app/views/issueDetails/streamline/sidebar/seerSectionCtaButton.tsx

```diff
@@ -185,7 +202,7 @@ export function SeerSectionCtaButton({
 
   return (
     <StyledButton
-      ref={openButtonRef}
+      to={seerLink}
```

Comment:
baseUrl is something else. use the current pathname from location
```suggestion
      to={{
    pathname: location.pathname,
    query: {
      ...location.query,
      seerDrawer: true,
    },
  }}
```

Reply:
unless it makes sense to send people to base of issue details i guess